File: app/src/main/java/de/fhkiel/temi/robogguide/database/Transfers.java
package de.fhkiel.temi.robogguide.database;

import com.j256.ormlite.field.DatabaseField;
import com.j256.ormlite.table.DatabaseTable;

@DatabaseTable(tableName = "transfers")
public class Transfers {
    @DatabaseField(generatedId = true)
    private int id;

    @DatabaseField
    private String title;

    @DatabaseField
    private int location_from;

    @DatabaseField
    private int location_to;
    //int users_id;


    public Transfers() {
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public int getLocation_from() {
        return location_from;
    }

    public void setLocation_from(int location_from) {
        this.location_from = location_from;
    }

    public int getLocation_to() {
        return location_to;
    }

    public void setLocation_to(int location_to) {
        this.location_to = location_to;
    }
}

File: app/src/main/java/de/fhkiel/temi/robogguide/database/Transfers.kt
package de.fhkiel.temi.robogguide.database

import com.j256.ormlite.field.DatabaseField
import com.j256.ormlite.table.DatabaseTable

/**
 * orm data class for transfers table
 *
 * @property id
 * @property title
 * @property location_from
 * @property location_to
 */
@DatabaseTable(tableName = "transfers")
data class Transfers (

    @DatabaseField(generatedId = true, columnName = "id")
    val id: Int,

    @DatabaseField(columnName = "title")
    val title: String,

    @DatabaseField(columnName = "location_from")
    val location_from: Int,

    @DatabaseField(columnName = "location_to")
    val location_to: Int,
)
{
    constructor(): this(0,"",0,0)
}
